packages/git-diff/lib/main.js

```diff
@@ -1,32 +1,52 @@
-const GitDiffView = require('./git-diff-view');
-const DiffListView = require('./diff-list-view');
+'use babel';
+
+import { CompositeDisposable } from 'atom';
+import GitDiffView from './git-diff-view';
+import DiffListView from './diff-list-view';
 
 let diffListView = null;
+let diffViews = null;
+let subscriptions = null;
```

Comment:
You can call CompositeDisposable in the global scope and avoid a lot of null check. It is only a simple `new Set`

Reply:
It's a single `null` check in the activate function that's only called once for the lifetime of the package. `null` occupies less memory than a set, so I figured I'd use that rather than leave extra memory hangin' around while the package is disabled.